Recipes/Recipes.Application/DTOs/Recipes/RecipeUpdateDTO.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Recipes.Application.DTOs.Recipes;
+
+public class RecipeUpdateDTO
+{
```

Comment:
I suppose the DTOs could also go to the Recipes.Public project. It is after all the public facing object representation format.

Reply:
Yup, seems logical